pkgs/development/libraries/gettext/default.nix

```diff
@@ -19,7 +19,16 @@ stdenv.mkDerivation rec {
     # fix reproducibile output, in particular in the grub2 build
     # https://savannah.gnu.org/bugs/index.php?59658
     ./0001-msginit-Do-not-use-POT-Creation-Date.patch
-  ];
+  ]
+    # An accidental inclusion in https://marc.info/?l=glibc-alpha&m=150511271003225&w=2
```

Comment:
Searching the glibc commits I can't find that this set eas ever included and was just something that other distros maintainer accidentally included in their build